### DISCUSSION TITLE
Homework 2

### DISCUSSION THREAD

**Discussion by avanickerson** [cooperrc/advanced-dynamics#8] [General]
When is HW_02 due? On HuskyCT, it says it is due tomorrow 1/17, but I figured that might be a typo. Just want to double check when the due date is!

**Comment by safdarmt** [comment]
I was wondering the same thing, it must be a typo.

**Comment by cooperrc** [comment]
Definitely a typo to start, thanks for your patience in the HuskyCT maintenance.

**Comment by travismalave** [comment]
Another question for HW 2 - HuskyCT shows the due date as 2/8 @ 12am. The syllabus says assignments are due at 11:59pm of the specified date. Is the intent that this assignment be due on 2/7 @ 11:59pm or 2/8 @ 11:59pm? Thanks!
